### DISCUSSION TITLE
How do I get active breakpoint in JavaScript?

### DISCUSSION THREAD

**Discussion by bangdragon** [unocss/unocss#2988] [General]
Like
const { isSm } = useBreakpoint('sm');
const { isMd } = useBreakpoint('md');
const { isLg } = useBreakpoint('lg');

**Comment by zyyv** [comment]
If you use https://vueuse.org/core/useBreakpoints/#usebreakpoints with UnoCSS, i think you just should get the theme of breakpoints in uno's config. like https://github.com/unocss/unocss/discussions/1908#discussioncomment-4371270.

**Reply by bangdragon** [reply]
I use sveltekit

**Reply by ryoppippi** [reply]
I use sveltekit too
